api/url.go

```diff
@@ -4,11 +4,13 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"github.com/howeyc/gopass"
```

Comment:
Can you take a look at https://godoc.org/golang.org/x/crypto/ssh/terminal and see what you think?  I used it on safe and it worked okay.  If this works better, I'd like to use it in safe. See https://github.com/jhunt/safe/blob/master/prompt/prompt.go